### DISCUSSION TITLE
More default values

### DISCUSSION THREAD

**Discussion by Mine181** [T8RIN/ImageToolbox#1407] [Ideas]
Add more default values that can be adjusted such as default image format, compression type, effort, quality, and speed.

**Comment by T8RIN** [comment]
The image format comes from the image, so it can't be adjusted, quality near the same

**Reply by Mine181** [reply]
Not even if you assume the image will always be converted? What other default values are planned to be added (if any)?

**Reply by T8RIN** [reply]
Yup, that's the way i created image retriever

**Reply by Mine181** [reply]
Alright, thanks for replying!

**Reply by T8RIN** [reply]
Other defaults, idk, have no idea at least now 👀 you can suggest smth, i will look if this possible
